Docs/ROADMAP.md

```diff
@@ -0,0 +1,328 @@
+build and sales period. After either a year has passed, or sales have dropped off to nothing, we can release the assets publicly so that this project can serve as a 
+starting point for the linux and Godot communities.
+
+**Advertising and Public Relations**
+In order to drive sales, we will want some form of advertising and public relations. This will involve posting promotional messaging to social media. The development of 
+promotional materials such as posters, art, and trailers will be critical to this game's marketability. I am not making any commitment to provide a marketing budget, but 
+it might very well be the case that I will chip in a couple hundred dollars in order to get some ads on websites once our product is polished.
+
+### Epics
+
+#### Characters
+
+We will need rigged models, textures, animations, and animation controllers, and voice-overs for all characters in this game.
+This should cover all three playable characters as well as a number of enemy characters.
+
+Each character should have a unique play style or niche in combat. This can partly be provided by a specific set of ICEPAWS stats, but should be reinforced with a set of 
+abilities that can be acquired and upgraded to modify this play style or niche.
+
+#### Items/abilities/upgrades
+
+We will need a set of items and abilities that can be used with all humanoid characters. If we have non-humanoid characters, we'll need abilities(ie biting, flying) that 
+may be unique to them.
+
+Each item or ability should have at least one possible upgrade that changes how it functions. These can come in the form of simply damage increases, or can introduce 
+entirely new functionality. (ie a crossbow that does less damage, but knocks the enemy back very far).
+
+#### Menus
+
+There are many menus outlined above in this document. We will need to build out and polish these menus where applicable, in order to deliver an intuitive and visually 
+appealing experience for the user.
+
+#### Music
+
+We will need a soundtrack that fits the game's mood and tone. We can use any music released CC-BY, or develop it outselves.
+This music will chiefly be broken into the following categories:
+- Opening menu music
+- Combat music
+- Victory music
+- Defeat music
+
+#### Sound effects
+
+We will need sound effects for just about every in-game action as well as audio cues when a player navigates through menus or
+earns points/experience/currency.
+
+#### Level design
+
+We will need to design levels for the arena encounters. All scenery and terrain will need to be modeled and textured.
+Each level should contain spawn points for players, enemies, and powerups. Interactive elements such as doors or destructable
+windows will also need to be developed in order to give
+
+#### User experience/balancing
+
+It will be critical to focus on delivering a good experience to our end users. To do this, we will need to allow external playtesting and respond
```

Comment:
I didn't see anything about builds/tooling, maybe for your team size simply exporting builds and putting it on an FTP server is enough but it might be useful to consider the workflow of how dev/alpha/beta/release builds get to the respective parties (or maybe that's outside the scope of this 🤷‍♂ )

Reply:
Fortunately, I don't think we'll need that many tools. Maintaining feature->develop->master branches, using release tags to capture specific milestones, and sharing binaries with external testers via something like google drive should get the job done.